src/NDSCart.cpp

```diff
@@ -26,6 +26,12 @@
 #include "melonDLDI.h"
 #include "FATStorage.h"
 #include "Utils.h"
+#include "RetroAchievements/RAClient.h"
+#include <rc_hash.h>
+
+namespace Config {
```

Comment:
The previous change in `NDS.cpp` introduces a namespace `melonDS::Config`, but this one introduces one just called `Config`. Why?